packages/mui-material-next/src/Slider/Slider.test.js

```diff
@@ -1437,4 +1441,48 @@ describe('<Slider />', () => {
       width: '10px',
     });
   });
+
+  describe('overlapping state', () => {
+    it('should apply thumb overlap classes', function test() {
+      if (/jsdom/.test(window.navigator.userAgent)) {
+        this.skip();
+      }
```

Comment:
```suggestion
```

I don't see why we should skip this in JSDOM. If it is not necessary let's remove it from all test, it will make the tests run faster.

Reply:
It's "necessary" because `getBoundingClientRect()` always returns `0, 0, 0, 0` in JsDOM (https://github.com/jsdom/jsdom/issues/653) and the element overlap calculation relies on it. In Karma, the tests work fine as `getBoundingClientRect()` returns the correct position.

What I could do is stub `getBoundingClientRect()`. Do you think that's better? I think the test is stronger without stubbing, but I would like to get your opinion.

Reply:
Ah I see, let's keep it as is, thanks for the clarification.